packages/gasket-plugin-swagger/index.js

```diff
@@ -74,6 +80,10 @@ module.exports = {
 
         await writeFile(target, content, 'utf8');
         gasket.logger.info(`Wrote: ${definitionFile}`);
+      } else {
+        gasket.logger.warning(
```

Comment:
This is a legitimate use case, and so should not warn here.

What we do need to warn on, is if `jsdoc` is configured but the swaggerSpec results are empty.

Reply:
@kinetifex am I correct in assuming that if the swaggerSpec results are empty, we also want to bypass https://github.com/godaddy/gasket/blob/UXP-4180/packages/gasket-plugin-swagger/index.js#L74-L82 ? Or should we simply warn and allow the rest of this block to run?

Reply:
Yea, that seems to make sense to skip the writes if no content.